src/triggers.js

```diff
@@ -617,7 +617,7 @@ export function maybeRunValidator(request, functionName) {
       });
   });
 }
-function builtInTriggerValidator(options, request) {
+async function builtInTriggerValidator(options, request) {
   if (request.master && !options.validateMasterKey) {
     return;
```

Comment:
Just a question here: you switched to async fn, does `builtInTriggerValidator` is correctly awaited elsewhere in the code ? (to avoid desync)

Reply:
As far as I remember it is only called returned in a promise.then, so it will still be resolved. I’ll check tho

Reply:
@Moumouls it is only called [here](https://github.com/parse-community/parse-server/blob/41a052c2c3bbb3676541e449af459441961539f1/src/triggers.js#L605) in a .then block, so it will be ok to be async, yes?